engine/src/main/java/org/pentaho/di/job/entries/trans/JobEntryTrans.java

```diff
@@ -1135,8 +1135,7 @@ public Result execute( Result result, int nr ) throws KettleException {
           //
           //trans = new Trans( transMeta, this );
```

Comment:
@rmansoor please update copyright header

Reply:
updated the header @ddiroma